analyzer-comments/elixir/rpn-calculator-output/close_in_after.md

```diff
@@ -0,0 +1,3 @@
+# close in after
+
+You can use the `after` block to exit gracefully whether the code in `try` was successful or not. Please use the `after` block to close `resource`.
```

Comment:
I'm not 100% sure if this sentence is grammatically correct. It feels to me like it's missing something like:
```suggestion
You can use the `after` block to exit gracefully regardless of whether the code in `try` was successful or not. Please use the `after` block to close `resource`.
```